tests/test_docs_admonition_inserter.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env python
+#
+# A library that provides a Python interface to the Telegram Bot API
+# Copyright (C) 2015-2023
+# Leandro Toledo de Souza <devs@python-telegram-bot.org>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser Public License for more details.
+#
+# You should have received a copy of the GNU Lesser Public License
+# along with this program.  If not, see [http://www.gnu.org/licenses/].
+
+import pytest
+
+import telegram.ext
+from docs.auxil.admonition_inserter import AdmonitionInserter
+
+
+@pytest.fixture(scope="session")
+def admonition_inserter():
+    return AdmonitionInserter()
+
+
+class TestAdmonitionInserter:
+    def test_admonitions_dict(self, admonition_inserter):
+        # there are keys for every type of admonition
+        assert len(admonition_inserter.admonitions) == len(admonition_inserter.ADMONITION_TYPES)
+
+        # for each type, there is at least one entry ({class/method: admonition text})
+        for admonition_type in admonition_inserter.ADMONITION_TYPES:
+            assert admonition_type in admonition_inserter.admonitions
+            assert len(admonition_inserter.admonitions[admonition_type].keys()) > 0
+
+            # keys are telegram classes
+            for cls in admonition_inserter.admonitions[admonition_type]:
+                # Test classes crop up in AppBuilder, they can't come from code being tested.
+                if "tests.conftest" in str(cls):
+                    continue
+
+                assert isinstance(cls, type)
+                assert "telegram" in str(cls), (
```

Comment:
maybe `str(cls).startswith('<class "telegram.")` or something like this? IIRC the format is always the same

Reply:
Done